database/postgres/index.js

```diff
@@ -1,7 +1,12 @@
 const { Pool, Client } = require('pg');
 const config = require('./config.js');
 
-const pool = new Pool(config);
+const pool = new Pool({
```

Comment:
Is this information input manually on the EC2 instance where you've deployed the service?